test/controller-test.js

```diff
@@ -0,0 +1,14 @@
+const { expect } = require("chai");
+
+const options = {method: 'GET'};
+
+describe("Controller contract", function () {
+  it("Deployment should allow user to deploy and acution for NFT", async function () {
+    const [owner, addr1, addr2] = await ethers.getSigners();
+
+    const NFTController = await ethers.getContractFactory("NFTController");
+
+    const hardhatToken = await NFTController.deploy();
+    
+  });
+});
```

Comment:
so this is just testing that the contract deploys, we want to test each function and make sure it does what we think it does - here's a good example in the hardhat docs: https://hardhat.org/tutorial/testing-contracts.html#full-coverage